draft-ietf-oauth-identity-chaining.md

```diff
@@ -133,6 +135,9 @@ The flow illustrated in Figure 1 shows the steps the client in trust Domain A ne
 ## Authorization Server Discovery
 This specification does not define authorization server discovery. A client MAY maintain a static mapping or use other means to identify the authorization server. The `authorization_servers` property in {{I-D.ietf-oauth-resource-metadata}} MAY be used.
 
+## Sender Constrained Tokens
+Either authorization server MAY issue sender constrained tokens. There are currently two options for sender constrained tokens: Mutually Authenticated TLS (mTLS) {{RFC8705}} and Demonstrating Proof of Possessions (DPoP) {{RFC9449}}.
```

Comment:
Readers may think this includes the assertion token. I believe limiting this to access tokens makes it more clear.
```suggestion
Either authorization server MAY issue sender constrained access tokens. There are currently two options for sender constrained tokens: Mutually Authenticated TLS (mTLS) {{RFC8705}} and Demonstrating Proof of Possessions (DPoP) {{RFC9449}}.
```

Reply:
also Possession is singular 
```suggestion
Either authorization server MAY issue sender constrained access tokens. There are currently two options for sender constrained tokens: Mutually Authenticated TLS (mTLS) {{RFC8705}} and Demonstrating Proof of Possession (DPoP) {{RFC9449}}.
```